app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
     height: "50vh",
     justifyContent: "center",
     textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      height: "100%",
+      margin: "0 auto",
+      width: "960px",
+    },
+  },
+  desktopNavigation: {
+    display: "flex",
+  },
+  introduction: {
+    color: "#ffffff",
+    textAlign: "left",
+    width: "742px",
+  },
+  loginLink: {
+    border: `1px solid ${theme.palette.primary.main}`,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    marginRight: theme.spacing(3),
+    padding: theme.spacing(1, 0),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
```

Comment:
use `margin` and `padding` rather than setting width explictly, or it's prone to breaking with theme changes.

Reply:
I was thinking about it but I used width on purpose so buttons are the same width - if you set margins/paddings then you will have button width depending on text length.

Reply:
I don't think horizontal buttons need to be the same width